src/components/simulation/tray-image.tsx

```diff
@@ -11,38 +11,52 @@ import "./tray-image.scss";
 // corner. This ends up being about 5 pixels for all tray objects. Technically it could be computed
 // by hand for ALL tray objects, but subtracting 5 pixels works pretty well in practice.
 const kOutlineOffset = 5;
+const kNonTrayPreviewHeight = 64;
```

Comment:
In playing with it I still felt like the non-tray images felt quite large. I think it would be worth trying a smaller height like 48.

Reply:
I agree.  I actually liked 28px the best - this makes the items the same size as the object when placed in the notebook.  However, Michael has 64px in the spec so I'll leave it for now, but hopefully it gets reduced once folks start using it.

Reply:
I didn't try it at other sizes, so I'm speculating, but I could see an argument that 28px would be too small, which is why I suggested something between 28 and 64. As you say, it'll be an easier conversation to have once there's something to play with.